version.xml

```diff
@@ -0,0 +1,8 @@
+<!-- This file may be overwritten by automation. -->
```

Comment:
The file may be interpreted as xml. e.g. build.sh/ps1 read the file to get values